crates/turbo-tasks/src/vc/resolved.rs

```diff
@@ -20,14 +20,24 @@ use indexmap::{IndexMap, IndexSet};
 
 use crate::{vc::Vc, RcStr};
 
-#[derive(Copy, Clone)]
```

Comment:
What was wrong with the derives here? At a glance they're equivalent? Also the PartialEq/Eq implementations look equivalent to what would've been derived?

Reply:
Derive would add `T: Clone` requirement, which we don't want.